src/Elements/Geometry/Color.cs

```diff
@@ -54,5 +54,46 @@ public override string ToString()
         {
             return $"R:{Red:0.00}, G:{Green:0.00}, B:{Blue:0.00}, A: {Alpha:0.00}";
         }
+
+        /// <summary>
+        /// Linearly interpolate between this color and the provide color.
```

Comment:
`provide` => `provided`. But since the word `other` is used, perhaps it should be that.